src/rules/types.ts

```diff
@@ -33,11 +34,17 @@ export interface AndDistinctRule extends Rule {
   reviewers: Reviewers[];
 }
 
+export interface FellowsRule extends Rule {
+  type: RuleTypes.Fellows;
+  minRank: number;
```

Comment:
does it make sense to make this consistent - either camelCase or snake for config or there's some purpose?

Reply:
I would prefer camelCase. Would it be fine if I make a second PR moments after this renaming `min_approvals` to `minApprovals`?